pkg/v1/providers/ytt/03_customizations/02_avi/ako-deployment.lib.yaml

```diff
@@ -88,6 +88,9 @@ data:
   logLevel: "WARN"
   deleteConfig: "false"
   serviceType:  NodePort
+#@ if data.values.AVI_NSXT_T1LR:
```

Comment:
Something useful to do would be to update
pkg/v1/providers/tests/clustergen/param_models/cluster_vsphere.model
to introduce the configuration of this configuration variable, like:

AVI_NSXT_T1LR: "", (and some other value that can legitimately me set)

so we can see in the clustergen output the effect of the overlay changes in responding these various values used.

Reply:
ack, added it